### ISSUE TITLE
Stuck on loading forever

### ISSUE THREAD

**Issue by Luxvao** [bitwarden/clients#8759]
### Steps To Reproduce

1. Install bitwarden through the arch repository
2. Log in
3. It loads forever

### Expected Result

It should actually load

### Actual Result

It gets stuck on loading.

### Additional Context

I've seen the others issues on this, but the only solution I could find was using snapstore. Is there any solution to use it when I installed it through the arch repository?

### Operating System

Linux

### Operating System Version

Arch Linux

### Installation method

Other

### Build Version

2024.4.0

### Issue Tracking Info

- [X] I understand that work is tracked outside of GitHub. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by trmartin4** [comment]
Hi @Luxvao!  This appears to be a duplicate of #8722.  I'll be closing this one, and please refer #8722 for any updates.  If you feel like this is a different issue please feel free to re-open.

**Comment by DrVektor** [comment]
> Hi @Luxvao! This appears to be a duplicate of #8722. I'll be closing this one, and please refer #8722 for any updates. If you feel like this is a different issue please feel free to re-open.

It's definitely a different subject. While the other two issues are the problem of adding new item. But here; there is no login and the loading circle goes on forever. Also the other threads are on ubuntu and this one is on arch linux. Please help. I have same issue.

**Comment by suevz** [comment]
> Hi @Luxvao! This appears to be a duplicate of #8722. I'll be closing this one, and please refer #8722 for any updates. If you feel like this is a different issue please feel free to re-open.

These are not the same things. I have the same problem as DrVektor and Luxvao.

**Comment by JIbald** [comment]
I am also experiencing the same issue on Arch Linux.
Even disabling a VPN does NOT help.
Also the VPN shouldn't be a problem in the first place I feel.
Thanks.

**Comment by kamaradclimber** [comment]
I face exactly the same issue on archlinux with the firefox extension.
No VPN.

**Comment by Exubitor** [comment]
New to Archlinux and stumbled upon the same issue, logging into Bitwarden works but it's loading with no end in sight (no VPN).

**Comment by ang-63** [comment]
Hi! 
I have today some problem with small difference. First of all i have another Arch Linux installation where i setup bitwarden password manager with no problem. This is a new instance with Arch+Hyprland if it mean something. I did install bitwarden with pacman and try to setup with login and master passowrd. I am absolutely sure that login and password was right. Btw i have 2 factor authentification and after password i did input code from google autenthificator as usual. No dice. Then i try to login with login with another device - something like that. I got request from my smartphone and oked it. Bitwarden begin to load data and stuck forever.

**Comment by kamaradclimber** [comment]
@trmartin4 (sorry to ping you), it seems the issue continues to affect people.
What kind of information should we provide to help resolution?

**Comment by hybridhavoc** [comment]
Version 2024.4.3 on Windows 11. I have not been able to use the Desktop application for at least a month now. Uninstall, reinstall, able to sign in but after authentication I only get the loading animation.

**Comment by SoldierCorp** [comment]
Same problem on Manjaro

**Comment by pjerem** [comment]
Same problem on OpenSUSE Tumbleweed since the latest system update.

**Comment by pjerem** [comment]
> Same problem on Manjaro

Same problem on OpenSUSE Tumbleweed since the latest system update.

> @trmartin4 : Hi @Luxvao! This appears to be a duplicate of #8722. I'll be closing this one, and please refer #8722 for any updates. If you feel like this is a different issue please feel free to re-open.

It's not the same problem, the desktop application isn't loading anymore : 
![image](https://github.com/bitwarden/clients/assets/72425/069db029-7e7f-4881-834f-6729b74bc899)

**Comment by Priviatech** [comment]
Same here on Ubuntu 22.04.4
.deb install
No VPN
Firefox add-on works perfectly on this host

```
15:09:50.446 › Error: L’adresse indiquée est vide
(node:10807) UnhandledPromiseRejectionWarning: TypeError: Cannot read properties of undefined (reading 'setContextMenu')
    at TrayMain.updateContextMenu (/opt/Bitwarden/resources/app.asar/main.js:88570:23)
    at MessagingMain.updateTrayMenu (/opt/Bitwarden/resources/app.asar/main.js:87998:28)
    at MessagingMain.onMessage (/opt/Bitwarden/resources/app.asar/main.js:87936:22)
    at MessagingMain.<anonymous> (/opt/Bitwarden/resources/app.asar/main.js:87924:155)
    at Generator.next (<anonymous>)
    at /opt/Bitwarden/resources/app.asar/main.js:87901:71
    at new Promise (<anonymous>)
    at messaging_main_awaiter (/opt/Bitwarden/resources/app.asar/main.js:87897:12)
    at IpcMainImpl.<anonymous> (/opt/Bitwarden/resources/app.asar/main.js:87924:83)
    at IpcMainImpl.emit (node:events:517:28)
(node:10807) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
15:09:50.519 › Error: L’adresse indiquée est vide
15:09:50.523 › Error: L’adresse indiquée est vide
15:09:50.574 › Error: Failed to decode access token: JWT must have 3 parts
```

**Comment by PedroBuffon** [comment]
Same on Debian 12, installed via .deb and same problem appears, idk what to do now. Waiting for a fix.

**Comment by aleoscar** [comment]
I also have this issue on Arch after installing bitwarden through pacman. When I uninstalled that and installed the appimage from the bitwarden website it works as expected, but it would be much nicer to not have to use the appimage.

**Comment by hybridhavoc** [comment]
> Version 2024.4.3 on Windows 11. I have not been able to use the Desktop application for at least a month now. Uninstall, reinstall, able to sign in but after authentication I only get the loading animation.

Version 2024.6.2 on Windows 11 is no longer running into the same problem for me.

**Comment by JIbald** [comment]
> I am also experiencing the same issue on Arch Linux. Even disabling a VPN does NOT help. Also the VPN shouldn't be a problem in the first place I feel. Thanks.

I no longer have this issue. Thank god.
Thanks.

**Comment by Priviatech** [comment]
> Same here on Ubuntu 22.04.4 .deb install No VPN Firefox add-on works perfectly on this host
> 
> ```
> 15:09:50.446 › Error: L’adresse indiquée est vide
> (node:10807) UnhandledPromiseRejectionWarning: TypeError: Cannot read properties of undefined (reading 'setContextMenu')
>     at TrayMain.updateContextMenu (/opt/Bitwarden/resources/app.asar/main.js:88570:23)
>     at MessagingMain.updateTrayMenu (/opt/Bitwarden/resources/app.asar/main.js:87998:28)
>     at MessagingMain.onMessage (/opt/Bitwarden/resources/app.asar/main.js:87936:22)
>     at MessagingMain.<anonymous> (/opt/Bitwarden/resources/app.asar/main.js:87924:155)
>     at Generator.next (<anonymous>)
>     at /opt/Bitwarden/resources/app.asar/main.js:87901:71
>     at new Promise (<anonymous>)
>     at messaging_main_awaiter (/opt/Bitwarden/resources/app.asar/main.js:87897:12)
>     at IpcMainImpl.<anonymous> (/opt/Bitwarden/resources/app.asar/main.js:87924:83)
>     at IpcMainImpl.emit (node:events:517:28)
> (node:10807) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
> 15:09:50.519 › Error: L’adresse indiquée est vide
> 15:09:50.523 › Error: L’adresse indiquée est vide
> 15:09:50.574 › Error: Failed to decode access token: JWT must have 3 parts
> ```

Problem solved with version 6.3
